content/rc/administration/setup/customize-flexible-plan.md

```diff
@@ -0,0 +1,123 @@
+
+| _Advanced option_ | _Description_ |
+|:---------|:-----------|
+| **Deployment CIDR** | The [CIDR](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing) subnet address for your deployment. Must not overlap other addresses used with your subscription.|
+| **Preferred Availability Zone(s)** | (_GCP only_) The [availability zone](https://cloud.google.com/compute/docs/regions-zones/#available) for your selected region.<br/><br/>If you choose *Select zone(s)*, you must choose at least one zone from **Zone Suffix**.  You can choose up to three preferred zones. |
+| **Persistent Storage Encryption** | (_AWS only_) Whether data is encrypted when stored to disk.  When selected, this mounts [encrypted EBS volumes](https://aws.amazon.com/ebs/faqs/#Encryption) to your instance. |
+
+When finished, choose **Next** to determine your size requirements.
+
+## Size tab
+
+The **Sizing** tab helps you specify the database, memory, and throughput for your subscription.
+
+When you first visit the **Sizing** tab, you're prompted to define a new database:
+
+{{<image filename="images/rc/flexible-create-sizing-first.png" width="75%" alt="Defining a database on the Sizing tab of the Create Custom Subscription screen." >}}{{< /image >}}
+
+| New database setting | Description |
+|:---------|:-----------|
+| **Name** | A custom name for your database (_required_) |
+| **Protocol** | _Redis_ or _Memcached_ |
+| **Memory Limit (GB)** | The size limit for the database, Specify small sizes as decimals of 1.0&nbsp;GB; example: `0.1` GB (minimum).| 
+| **Throughput By** | Identifies how throughput is measured for the database, either operations per second (_Ops/sec_) or _Number of shards_. |
+| **Modules** | Identifies a module used by the database.  Choose from [RedisSearch&nbsp;2](#), [RedisGraph](#), [RedisBloom](#), or [RedisTimeSeries](#).<br/><br/>If you select RedisSearch 2, you also need to specify a value for **Number of Documents**.  This defines the maximum internal array size ([MAXDOCTABLESIZE](https://oss.redislabs.com/redisearch/Configuring/?_ga=2.155176508.524468484.1612194154-499260268.1607530891#maxdoctablesize)).|
+| **Replication** | Enables an in-memeory replica, which in turn enables real-time failover, but doubles memory consumption. |
+| **Data Persistence** | Defines the data persistence policy, if any. See [Database Persistence]({{< relref "/rs/concepts/data-access/persistence.md" >}}) |
+| **OSS Cluster API support** | Allows client to use the OSS Cluster API to access the database. |
+| **Ops/sec** | The operations required per second (when **Throughput By** is set to _Ops/sec_) |
+| **Number of Shards** | The number of shards set aside for the database (when **Throughput By** is set to _Number of Shards_) |
+
+When finished, click the **Save** button to add the database to your subscription.
+
+When you have at least one defined database, the **Sizing** tab shows a list of your defined databases and provides to maintain them.
```

Comment:
```suggestion
When you have at least one defined database, the **Sizing** tab shows a list of your defined databases and provides (missing word?) to maintain them.
```
Looks like there is a word missing here?

Reply:
Resolved